docs/calicoctl/container.md

```diff
@@ -0,0 +1,159 @@
+
+# User guide for 'calicoctl container' commands
+
+This sections describes the `calicoctl container` commands.
+
+These commands can be used to manage Calico networking for Docker containers.
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl container' commands
+
+Run
+
+    calicoctl container --help
+
+to display the following help menu for the calicoctl container commands.
+
+```
+
+Usage:
+  calicoctl container <CONTAINER> ip (add|remove) <IP> [--interface=<INTERFACE>]
+  calicoctl container <CONTAINER> endpoint show
+  calicoctl container <CONTAINER> profile (append|remove|set) [<PROFILES>...]
+  calicoctl container add <CONTAINER> <IP> [--interface=<INTERFACE>]
+  calicoctl container remove <CONTAINER>
+
+Description:
+  Add or remove containers to calico networking, manage their IP addresses and profiles.
+  All these commands must be run on the host that contains the container.
+
+Options:
+  --interface=<INTERFACE>  The name to give to the interface in the container
+                           [default: eth1]
+
+```
+
+## calicoctl container commands
+
+### calicoctl container <CONTAINER> ip add <IP> 
+
+This command allows you to add an IP address to a container that is already
```

Comment:
I *personally* think it would be nice to go into a little more detail for some of these commands.  For example, with these container commands, what does it mean to be added to Calico networking.  What do these commands actually do?